src/libraries/System.Text.Json/src/System/Text/Json/JsonConstants.cs

```diff
@@ -42,10 +42,10 @@ internal static partial class JsonConstants
         public static ReadOnlySpan<byte> NegativeInfinityValue => "-Infinity"u8;
 
         // Used to search for the end of a number
-        public static ReadOnlySpan<byte> Delimiters => new byte[] { ListSeparator, CloseBrace, CloseBracket, Space, LineFeed, CarriageReturn, Tab, Slash };
+        public static ReadOnlySpan<byte> Delimiters => ",}] \n\r\t/"u8;
```

Comment:
I'm not sure this one is "better", it's certainly smaller but also a lot harder to understand what's being included in here and easier to miss things as a potential typo/etc

Reply:
I went back and forth on this one, but ultimately I found it easier to understand what was actually included, rather than needing to refer back to consts for things like ListSeparator to know it was a comma. I don't feel strongly about it, though. I was tipped in favor of it by the IDE analyzer recommending it, and should we decide to enable it, thinking it better to just change it rather than suppress around it.